src/main/java/it/pagopa/pn/service/desk/action/impl/ValidationOperationActionImpl.java

```diff
@@ -114,6 +83,39 @@ public void execute(String operationId) {
                 .block();
     }
 
+    private Mono<Void> checkValidationFlow(PnServiceDeskOperations operation, PnServiceDeskAddress addres) {
+        return getIuns(operation.getRecipientInternalId())
+                .collectList()
+                .doOnNext(responsePaperNotificationFailed -> {
+                    log.debug("listOfIuns = {}, List of iuns retrivied", responsePaperNotificationFailed);
+                    operation.setErrorReason(null);
+                    log.debug("errorReason = {}, Setting to null errorReason into entityOperation", operation);
+                    updateOperationStatus(operation, OperationStatusEnum.VALIDATION);
+                })
+                .flatMapMany(Flux::fromIterable)
+                .flatMap(iun -> {
+                    log.debug("iun = {}, Get attachment from iun", iun);
+                    return getAttachmentsFromIun(operation, iun);
+                })
+                .flatMap(pnServiceDeskAttachments -> getFileKeyFromAttachments(pnServiceDeskAttachments))
```

Comment:
utilizziamo il method reference: .flatMap(this::getFileKeyFromAttachments)

Reply:
fixato